Makefile

```diff
@@ -23,7 +23,12 @@ $(EMBEDDED_STATIC): static/kubeapps-objs.yaml
 	$(GO) generate
 
 static/kubeapps-objs.yaml:
-	KUBECFG_JPATH=./manifests/lib:./manifests/vendor/kubecfg/lib:./manifests/vendor/ksonnet-lib kubecfg show -V VERSION=$(VERSION) manifests/kubeapps.jsonnet > static/kubeapps-objs.yaml
+	KUBEAPPS_VERSION=$$(git describe --tags --abbrev=0) ;\
```

Comment:
this ensures we get the latest tag (e.g. currently v0.6.0). again, ideally we use the latest tag to get any changes in components in master since the last version, but at least this makes it a bit easier for now.